x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -171,7 +171,7 @@ func createCloudWatchEvents(getMetricDataResults []cloudwatch.MetricDataResult,
 			continue
 		}
 		labels := strings.Split(*output.Label, " ")
-		mapOfMetricSetFieldResults[labels[1]] = fmt.Sprint(output.Values[0])
+		mapOfMetricSetFieldResults[labels[1]] = fmt.Sprint(output.Values[len(output.Values)-1])
```

Comment:
This sounds like a bug, should we add a changelog entry or was it never shipped like this?

Can it be that `len(output.Values) == 0`. If yes, this could cause troubles I think.

Reply:
Yeah it can be and there is a check on line 170 https://github.com/elastic/beats/pull/11142/files/39e7026612a5749109b53cb5ae648648c137c2ff#diff-019d1f0f86b061ba4cc82e0fa0c1e30aR170 to avoid the troubles 😄 I will add a changelog for this! Thanks!